src/verifiers/documentStatus/tokenRegistryStatus/openAttestationEthereumTokenRegistryStatus.test.ts

```diff
@@ -110,32 +110,33 @@ describe("openAttestationEthereumTokenRegistryStatus", () => {
           network: "ropsten",
         }
       );
-      expect(fragment).toStrictEqual([
-        {
-          name: "OpenAttestationEthereumTokenRegistryStatus",
-          type: "DOCUMENT_STATUS",
-          data: {
-            details: [
-              {
-                address: "0xabcd",
-                minted: false,
-                reason: {
-                  code: 2,
-                  codeString: "CONTRACT_ADDRESS_INVALID",
-                  message: "Contract address 0xabcd is invalid",
+      expect(fragment).toMatchInlineSnapshot(`
+        Array [
+          Object {
+            "data": Object {
+              "details": Array [
+                Object {
+                  "minted": false,
```

Comment:
I'm not sure what happened but previously the address was available in the details (it's also available in document store fragment)

Reply:
Well.... you had a common error emitter now, so it doesnt have the context of the addresses....
Also it's not important to have address i guess since we only allow one token registry now.

Reply:
It's shown in document store => https://github.com/Open-Attestation/oa-verify/pull/132/files#diff-edaba66bd4a52c0a21fcee6b1cc9a16f8183f6f18053fa374f1b110c7c1d2de9R109 :thinking:

Reply:
Fixed

Reply:
maybe a different scenario :thinking: